onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -363,15 +361,19 @@ class PlannerImpl {
         if (node_input->Exists())
           UseCount(node_input->Name())++;
       }
-
       // Identify where each output of this node should be allocated.
       // This is determined by the opkernel bound to the node.
+      //TODO: does 'pnode' equal to 'node' ? Can we remove the following line?
       auto node = graph_viewer_.GetNode(step.node_index);
       ORT_ENFORCE(nullptr != node);
```

Comment:
I would expect it to be the same as I don't believe step.node_index changes here.